crates/dbsp/src/circuit/runtime.rs

```diff
@@ -357,31 +320,16 @@ impl Runtime {
         let workers = config.layout.local_workers();
         let nworkers = workers.len();
 
-        let storage: Result<StorageLocation, DbspError> = config.storage.clone().map_or_else(
-            // Note that we use into_path() here which avoids deleting the temporary directory
-            // we still clean it up when the runtime is dropped -- but keep it around on panic.
-            || {
-                if config.init_checkpoint != Uuid::nil() {
-                    return Err(DbspError::Storage(StorageError::CheckpointNotFound(
-                        config.init_checkpoint,
-                    )));
-                }
-                Ok(StorageLocation::Temporary(
-                    tempfile::tempdir().unwrap().into_path(),
```

Comment:
This did things slightly differently with the temp dir I wonder if we dont do that anymore